install/pbench_agent_install.yml

```diff
@@ -0,0 +1,107 @@
+# ansible-playbook pbench_agent_install.yml
+
+- hosts: compute
+  gather_facts: yes
+  tasks:
+    - name: install latest epel and perl-Data-UUID
+      yum:
+        name: "{{ epel_repo.el7 }}"
+        state: present
+      become: yes
+      loop:
+        - "{{ epel_repo.el7 }}"
+        - perl-Data-UUID
```

Comment:
Why do you have to install `perl-Data-UUID`?

Reply:
These will install perl-Data-UUID after the epel is installed on computes as by default the yum repos are disabled

Reply:
But why is this needed explicitly?  The pbench-agent RPM should have dependencies that take care of this.